app/models/health/participation_form.rb

```diff
@@ -35,6 +35,7 @@ class ParticipationForm < HealthBase
     scope :valid, -> do
       parent_ids = Health::ParticipationFormFile.where.not(parent_id: nil).select(:parent_id)
       where.not(location: [nil, '']).
```

Comment:
Is this a problem?  Didn't we remove the location field from the form?

Reply:
We removed the location field from the UI, but any forms that were submitted with a location value, we left as they were... So, this deals with the legacy case.